airbyte-integrations/connectors/source-zendesk-talk/source_zendesk_talk/schemas/ivr_menus.json

```diff
@@ -16,35 +16,6 @@
     },
     "name": {
       "type": ["null", "string"]
-    },
-    "routes": {
```

Comment:
was this schema just wrong?

Reply:
it wasn't but we have these routes as separate stream and also as nested object in ivrs stream